generator/config.js

```diff
@@ -42,6 +42,11 @@ exports.mergeConfigs = function mergeConfigs(args, config) {
     noReact: !!args["--noReact"] || config.noReact,
     namingConvention: args["--dontRenameModels"]
       ? "asis"
-      : config.namingConvention
+      : config.namingConvention,
+    header:
+      args["--header"] ||
+      Object.keys(config.header)
```

Comment:
I think this will cause issues if there aren't headers in a config.

`TypeError: Cannot convert undefined or null to object`

Reply:
Good call, this was done pretty quickly.  Will revisit.  Thanks.